src/DrTests-Tests/DrTestsUITest.class.st

```diff
@@ -24,114 +30,164 @@ DrTestsUITest >> testBrowseButton [
+			
+	self
+		assert: drTestsUI statusLabel label
+		equals: ('{1}: {2}' format: { (drTestsUI  dateAndTimeString ). status }).
+]
+
+{ #category : #tests }
+DrTestsUITest >> testInitialWindowTitleIsInitialPluginWindowTitle [
 	
+	self
+		assert: drTestsUI title
+		equals: 'Dr Tests - ' , drTestsUI currentPlugin pluginName.
 ]
 
 { #category : #tests }
 DrTestsUITest >> testMultipleSelectingPackagesWillUpdateTheClassesList [
-	| currentPluginSelected  randomPackage newPackagesSelected classesList|
+	| currentPluginSelected randomPackage newPackagesSelected classesList packageSelected |
 	currentPluginSelected := drTestsUI pluginsDropList selectedItem.
-   newPackagesSelected  := OrderedCollection new.
-	randomPackage:=(drTestsUI packagesList listItems) anyOne.
+	newPackagesSelected := OrderedCollection new.
+	packageSelected := drTestsUI packagesList listItems anyOne.
+	randomPackage := (drTestsUI packagesList listItems
+		\ {packageSelected}) anyOne.
+	newPackagesSelected add: packageSelected.
 	newPackagesSelected add: randomPackage.
-	drTestsUI whenPackagesSelectionChanged: newPackagesSelected   .
-	classesList := (currentPluginSelected new selectClasesFrom: newPackagesSelected ).
-	
-	self assertCollection: drTestsUI classesList listItems hasSameElements: classesList flattened .
-	
-	
-	
-"	newPluginSelected := drTestsUI pluginsDropList selectedItem.
-	
-	self deny: currentPluginSelected equals: newPluginSelected.
-	
-	self assertCollection: drTestsUI packagesList listItems hasSameElements: newPluginSelected new selectPackages."
+	drTestsUI whenPackagesSelectionChanged: newPackagesSelected.
+	classesList := currentPluginSelected new
+		selectClasesFrom: newPackagesSelected.
+	self
+		assertCollection: drTestsUI classesList listItems
+		hasSameElements: classesList flattened
+
+
 ]
 
 { #category : #tests }
 DrTestsUITest >> testNoPackageSelected [
-	| currentPluginSelected  currentPackage newPackageSelected classesList|
-	currentPluginSelected := drTestsUI pluginsDropList selectedItem.
-	
-	currentPackage := drTestsUI packagesSelected.
-	classesList := drTestsUI classesList getList.
-	
-	
-	self assertCollection: drTestsUI classesList listItems hasSameElements: #().
-	currentPackage:=(drTestsUI packagesList listItems ) anyOne.
-	
+	self
+		assertCollection: drTestsUI classesList listItems
+		hasSameElements: #()
+]
+
+{ #category : #tests }
+DrTestsUITest >> testPluginsAreConfiguredPluginsOnly [
+	self
+		assert: drTestsUI pluginsDropList listItems asArray
+		equals: plugins
+]
+
+{ #category : #tests }
+DrTestsUITest >> testResultButtonHelpIsCurrentPluginResultButtonHelp [
 	
-	drTestsUI pluginsDropList setSelectedItem: (drTestsUI pluginsDropList listItems \ { currentPluginSelected }) anyOne.
+	self
+		assert: drTestsUI browseButton help
+		equals: drTestsUI currentPlugin resultButtonHelp
+]
+
+{ #category : #tests }
+DrTestsUITest >> testResultButtonIsCurrentPluginResultButton [
```

Comment:
This test is empty, so it looks green but in fact does nothing.

Should either be implemented or removed.